addons/hr_org_chart/static/src/js/hr_org_chart.js

```diff
@@ -75,7 +75,8 @@ var FieldOrgChart = AbstractField.extend({
             }));
         }
         else if (!this.employee) {
-            this.employee = this.recordData.id
+            // the widget is either dispayed in the context of a hr.employee form or a res.users form
+            this.employee = this.recordData.employee_ids ? this.recordData.employee_ids.res_ids[0] : this.recordData.id;
```

Comment:
Be careful with this code, in js an empty list is not falsy
eg : 
``` javascript
var a = [];
if (a) {
    console.log('Hello');
}
```
Will output Hello

Reply:
`this.recordData.employee_ids` is an object and is undefined if the org chart is displayed from a `hr.employee` form view. It should be good as it is.
I will add `this.recordData.employee_ids === undefined` to make it explicit.